### DISCUSSION TITLE
DAX language

### DISCUSSION THREAD

**Discussion by hassan75016** [github-linguist/linguist#5904] [Classification] [answered]
Hello - it seems like GitHub linguist does not recognize DAX language. Syntax is properly formatted and color coded in VS code however, GitHub code preview always shows it as plain text. The files are properly saved as .dax files. Lengthy DAX code in plain text makes it very difficult for contributors to understand. Is there a workaround available or the possibility to add support for this language? 
Screenshot attached for reference.
Thanks. 

![dax](https://user-images.githubusercontent.com/51833534/169692137-96f46e98-6e08-4aff-aa5b-bd7be43263e6.png)

**Comment by lildude** [comment, marked as answer]
> GitHub linguist does not recognize DAX language

Correct, but you can change this by following the details in [CONTRIBUTING.md](https://github.com/github/linguist/blob/master/CONTRIBUTING.md). If you don't have the time or experience, you can open a new [issue requesting](https://github.com/github/linguist/issues/new/choose) this and someone else will add it at some point in the future.

There isn't a workaround short of using an override to identify the files as some other language.

**Reply by hassan75016** [reply]
Thanks - I've raised an issue/request.

**Reply by lildude** [reply]
Why didn't you use the template? Please modify your issue to use the new language template and tags.
